omod/src/main/webapp/resources/htmlforms/PatientReferral.xml

```diff
@@ -289,6 +343,10 @@
                                             <encounterLocation default="629d78e9-93e5-43b0-ad8a-48313fd99117"
                                                                order="629d78e9-93e5-43b0-ad8a-48313fd99117"/>
                                         </span>
+                        <span style="display:none;">
+                            <obs id="99160" conceptId="99160"/>
```

Comment:
@dbaluku can you use ids that make sense, as I have to look up these IDS to understand what they are referring to

Reply:
Done